bool published_frequency
bool depth_limit
bool batteries_limit
bool depressurization
bool seafloor
